integration/localstack/ls_tmp/snakeoil.conf

```diff
@@ -0,0 +1,18 @@
+[req]
+default_bits = 2048
+prompt = no
+default_md = sha256
+x509_extensions = v3_req
+distinguished_name = dn
+[dn]
+C = US
+ST = VA
+L = Virginia
+O = CloudwatchAgent
+emailAddress = sethlevi@amazon.com
```

Comment:
Do we need this? I'm not convinced we need an email address for a signed SSL cert

Reply:
duplicate